spec/lib/axe/matchers/be_accessible_spec.rb

```diff
@@ -5,14 +5,13 @@ module Axe::Matchers
   describe BeAccessible do
     let(:a11y_check) { spy('a11y_check', call: audit) }
```

Comment:
We should remove a11y_check

Reply:
I can do that as a separate PR. It's calling `axe.run` internally, not `axe.a11yCheck`. But we can update the API method for a major release.